docs/common/types.ts

```diff
@@ -0,0 +1,35 @@
+export interface Slugger {
```

Comment:
Should we move this to a "root" folder called `types`? Seems like that's more used generally speaking, and allows us to add files targeting specific libraries or pieces of code I think.

Reply:
And in this case, we should add [`@types/github-slugger`](https://www.npmjs.com/package/@types/github-slugger) maybe?

Reply:
I haven't noticed that `@types/github-slugger` exist, I'll check it out.
Moving the file to separate folder would be a good idea also.